pkg/qdr/qdr.go

```diff
@@ -836,9 +838,68 @@ func (a *BridgeConfig) Difference(b *BridgeConfig) *BridgeConfigDifference {
 		HttpConnectors: a.HttpConnectors.Difference(b.HttpConnectors),
 		HttpListeners:  a.HttpListeners.Difference(b.HttpListeners),
 	}
+
+	result.AddedSslProfiles = checkAddedSslProfiles(result)
+	result.DeletedSSlProfiles = checkDeletedSslProfiles(result, a, b)
+
 	return &result
 }
 
+func checkAddedSslProfiles(difference BridgeConfigDifference) []string {
+
+	var addedSslProfiles []string
+
+	for _, httpConnector := range difference.HttpConnectors.Added {
+		if len(httpConnector.SslProfile) > 0 {
+			addedSslProfiles = append(addedSslProfiles, httpConnector.SslProfile)
+		}
+	}
+
+	for _, httpListener := range difference.HttpListeners.Added {
+		if len(httpListener.SslProfile) > 0 {
```

Comment:
Is it possible that the httpListener or httpConnector will share the same sslProfile?
If so, don't you need to validate if the sslProfile is already in the slice before adding?
Or each one uses its own sslProfile?